ui/cypress/integration/tests.js

```diff
@@ -6,6 +6,7 @@ describe("Basic tests", () => {
     cy.contains("Datasets");
 
     cy.get("button[id=insert-cohort]").click();
+    cy.get("#text").type("New Cohort");
```

Comment:
I just noticed this doesn't have a `.click()` on it. What is it actually doing?

Reply:
Because we change the text blank as empty while we do not assign any initialText. Like create a new Cohort. 
So we must type the name "New Cohort" in the dialog. on the line 10 it has `.click()` to create the new cohort.

Reply:
Oh, I see. I was thinking it meant like the element type or something but it actually types those characters in.